### PR TITLE
Hw1

### PR DESCRIPTION
**Pull request by yatskivbohdan**

### CODE REVIEW

**Comment by yatskivbohdan** [comment]
@pavlo-berezin

File: src/Footer/fs-footer-contact-bar.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+
+import InstaLogo from '../assets/footer-icons/insta-logo.png'
+import FacebookLogo from '../assets/footer-icons/facebook-logo.png'
+import InLogo from '../assets/footer-icons/linkedin-logo.png'
+import GitLogo from '../assets/footer-icons/github-logo.png'
+
+
+const Contact = ({name, href, logo}) => (
```
Comment (pavlo-berezin) [inline src/Footer/fs-footer-contact-bar.js:10]: 1 component per file.

File: src/Footer/fs-footer-contact-bar.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```
Comment (pavlo-berezin) [inline src/Footer/fs-footer-contact-bar.js:1]: Please fix indentation in this file

File: src/Footer/fs-footer.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import './fs-footer.css'
+
+import ContactBar from './fs-footer-contact-bar'
+
+
+export default class Footer extends React.Component {
+    constructor() {
```
Comment (pavlo-berezin) [inline src/Footer/fs-footer.js:9]: No need of empty constructor.

File: src/Header/fs-header-menu-item.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const LeagueLogo =({logo}) => (
```
Comment (pavlo-berezin) [inline src/Header/fs-header-menu-item.js:3]: One component per file

File: src/Header/fs-header.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+.header-menu-league {
+    display: flex;
+    margin:0;
+    align-items: center;
+    justify-content: flex-start;
+    list-style-type: none;
+    overflow: hidden;
+}
+
```
Comment (pavlo-berezin) [inline src/Header/fs-header.css:39]: For dividing content 1 empty line is enough.

File: src/Header/fs-header.css [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+.header-contact-bar {
+    height: 100%;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+}
+
+.search-container {
```
Comment (pavlo-berezin) [inline src/Header/fs-header.css:80]: Please try having 1 css file per 1 component, with all related css there.

File: src/MainBody/fs-main-body.js [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+                    </Switch>
+                </Router>
+            </div>
+            );
+              
+    }
+}
+
+
+/*<div className="pie-chart">
```
Comment (pavlo-berezin) [inline src/MainBody/fs-main-body.js:45]: Please remove commented code.

**Comment by pavlo-berezin** [comment]
Almost all files have some problems with indentation, please fix this.

File: src/Header/fs-header-menu-item.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+const LeagueLogo =({logo}) => (
+    <li className="header-menu-league-part">
+        <img className="league-logo" src={logo}></img>
+    </li>
+);
+
+const LeagueName =({text, href}) => (
+    <li className="header-menu-league-part">
+        <a className="league-name" href={href}>{text}</a>
```
Comment (pavlo-berezin) [inline src/Header/fs-header-menu-item.js:11]: Use react-routers `Link` or `NavLink` instead of `a`

File: src/MainBody/fs-main-body-standings.js [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+      id: id,
+      error: null,
+      isLoaded: false,
+      rows: [],
+    };
+  }
+
+componentDidMount() {
+    fetch('http://api.football-data.org/v2/competitions/' + this.state.id + '/standings', { 
+      method: 'get',
```
Comment (pavlo-berezin) [inline src/MainBody/fs-main-body-standings.js:38]: 'get' is by default, no need to specify.

**Pull request merged**